docker/boot.sh

```diff
@@ -20,16 +20,16 @@ then
   python --version | sed 's/^/    /'
   command -v python | sed 's/^/    /'
 
-  APP_CONFIG=/home/scu/host-dev.yaml
+  # APP_CONFIG=/home/scu/host-dev.yaml
```

Comment:
All commented code should be justified with a note ``NOTE: commented for this/that reason`` 
This way we try to avoid having the code filled with forgotten pieces of commented code :-)